bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/model/svn/SVNMergeUnit.java

```diff
@@ -668,32 +670,28 @@ public Path convertSvnDiffToPath(final SvnDiff diff) {
 	private String getCommitInfo() {
 		LogUtil.entering();
 
-		long revStart = getRevisionStart();
-		long revEnd = getRevisionEnd();
-
-		if (revStart < 1 || revEnd < 1) {
+		long revision = getRevisionEnd();
+		if (revision < 1) {
 			LOGGER.severe(() -> String.format("Revision range is invalid. mergeUnit=%s", this)); //$NON-NLS-1$
 			return LogUtil.exiting(null);
 		} else {
-			StringBuilder commitMessage = new StringBuilder();
-			List<SvnLog> logEntries = getLogEntries(getUrlSource(), revStart, revEnd);
-
-			for (SvnLog logEntry : logEntries) {
-				commitMessage.append(String.format(COMMIT_MESSAGE_MERGE_PREVIOUS, logEntry.getRevision(),
-						logEntry.getAuthor(), logEntry.getDate(), logEntry.getMessage()));
+			Optional<SvnLog> logEntry = getLogEntries(getUrlSource(), revision);
+			if (logEntry.isPresent()) {
+				SvnLog entry = logEntry.get();
+				return LogUtil.exiting(String.format(COMMIT_MESSAGE_MERGE_PREVIOUS, entry.getRevision(),
+						entry.getAuthor(), entry.getDate(), entry.getMessage()));
+			} else {
+				return LogUtil.exiting(null);
 			}
-
-			return LogUtil.exiting(commitMessage.length() > 0 ? commitMessage.toString() : null);
 		}
 	}
 
-	private List<SvnLog> getLogEntries(String url, long revisionStart, long revisionEnd) {
-		final ISvnClient client = E4CompatibilityUtil.getApplicationContext().get(ISvnClient.class);
+	private Optional<SvnLog> getLogEntries(String url, long revision) {
```

Comment:
Now this should be called `getLogEntry` instead, as does not return any longer a list.

Reply:
Fixed